app/api/schema/event_role_permissions.py

```diff
@@ -0,0 +1,39 @@
+from marshmallow_jsonapi import fields
+from marshmallow_jsonapi.flask import Schema, Relationship
+
+from app.api.helpers.utilities import dasherize
+from utils.common import use_defaults
+
+
+@use_defaults()
+class EventsRolePermissionSchema(Schema):
+    """
+    API Schema for Permission Model
+    """
+
+    class Meta:
+        """
+        Meta class for Notification API schema
+        """
+        type_ = 'events-role'
+        self_view = 'v1.events_role_detail'
+        self_view_kwargs = {'id': '<id>'}
+        self_view_many = 'v1.events_role_list'
+        inflect = dasherize
+
+    id = fields.Str(dump_only=True)
+    can_create = fields.Boolean(default=False)
+    can_read = fields.Boolean(default=False)
+    can_update = fields.Boolean(default=False)
+    can_delete = fields.Boolean(default=False)
+    role = Relationship(attribute='role',
+                        self_view='v1.role_detail',
+                        self_view_kwargs={'id': '<id>'},
+                        schema='RoleSchema',
```

Comment:
Related view and related view kwargs ? Aren't they required ? Have you tested every endpoint if it is working or not ?